assets/locales/en.json

```diff
@@ -150,12 +151,15 @@
       "header": {
         "success": "Success",
         "canceled": "Cancelled",
-        "notSatisfiable": "Cancelled"
+        "not_satisfiable": "Cancelled"
       },
       "text": {
         "success": "Your data is disclosed to {otherParty}",
+        "success_signature": "You signed the request of {otherParty} with your data",
```

Comment:
Same here: maybe remove the 'with your data' part here and the other occurrences below?